udemy_exercises/cell_phone/Contact.java

```diff
@@ -1,7 +1,50 @@
-package udemy_exercises.cell_phone;
+package cell_phone;
 
 class Contact {
+	
+private String name;
+private String phoneNumber;
 
+    public Contact(String name, String phoneNumber) {
+        this.name = name;
+        this.phoneNumber = phoneNumber;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public void setPhoneNumber(String phoneNumber) {
+        this.phoneNumber = phoneNumber;
+    }
+    
+    public Contact saveContact(String name, String phoneNumber) {
+    	return new Contact(name, phoneNumber);
+    }
+    
+    public void modifyContact(String name, String phoneNumber) {
+    	this.name = name;
+    	this.phoneNumber = phoneNumber;
+    }
+    
+    public boolean searchContact(String name) {
+    	return this.name.equals(name);
+    }
+    
+    public void deleteContact(String name) {
```

Comment:
Does this really "delete" the contact?  Or just delete the properties of the contact?  Should 'delete' even be a responsibility of the 'Contact' class?